lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Lets just do that, I don't think documentation is needed, it is in the Sequelize docs.